### PR TITLE
Moved Customiser's CSS into theme files

### PR DESCRIPTION
**Pull request by vagelisp**
I converted the Customiser's CSS rules into SCSS and moved them to theme files.
I also added a file with the original code from the Customiser.

Related to #937

### CODE REVIEW

File: wp-content/themes/pub/wporg-learn-2020/css/components/_card.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
+.card {
+	padding: 1.75rem;
+}
```
Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2020/css/components/_card.scss]: This padding is declared below on line 11. The bit we're missing which affects the course cards on the home page is 

```
body:not(.single):not(.search) .site-main .card.post.course {
	padding: 1.75rem;
}
```

Screen Shot 2022-10-18 at 11 11 10 AM

Doesn't feel quite right for these styles to be added here, as it's an override for the sensei styles, so a comment is probably necessary, maybe:

```suggestion
// Override Sensei course styles for cards on home page
.body:not(.single):not(.search) .site-main .card.post.course {
	padding: 1.75rem;
}
```

File: wp-content/themes/pub/wporg-learn-2020/css/customiser.css [hunk 2]
```diff
@@ -0,0 +1,159 @@
+thead th {
```
Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2020/css/customiser.css]: I think we should delete this file. We have a record of the contents on the issue

File: wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss [hunk 3]
```diff
@@ -1,31 +1,175 @@
...
+    }
+
+    .entry {
+      display: none;
+    }
+  }
+}
+
+.sensei {
+  .course {
```
Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss]: I think these nested selectors (`.course`, `.lesson`, `.quiz`) should be nested under the existing `.sensei` selector on line 1

Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss:34]: I think these nested selectors (`.course`, `.lesson`, `.quiz`) should be nested under the existing `.sensei` selector on line 1

Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss:34]: I think these nested selectors (`.course`, `.lesson`, `.quiz`) should be nested under the existing `.sensei` selector on line 1

File: wp-content/themes/pub/wporg-learn-2020/css/customiser.css [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+}
+
+details summary {
+	display: inline-block;
+	font-weight: bold;
+	cursor: pointer;
+	line-height: 2.8;
+}
+
+/* Details summary block END */
```
Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2020/css/customiser.css]: We're missing this block unless I'm mistaken?

**Review by adamwoodnz — CHANGES_REQUESTED** [review]
Thanks for taking care of this @vagelisp!

A few comments inline 🙂

**Comment by vagelisp** [comment]
Thanks for the review @adamwoodnz 
I ll get these fixed in the next couple of days.

File: wp-content/themes/pub/wporg-learn-2020/css/components/_card.scss (hunk 1 above)
Comment (vagelisp) [reply wp-content/themes/pub/wporg-learn-2020/css/components/_card.scss]: That should be fixed now

File: wp-content/themes/pub/wporg-learn-2020/css/customiser.css (hunk 2 above)
Comment (vagelisp) [reply wp-content/themes/pub/wporg-learn-2020/css/customiser.css]: I removed the file

File: wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss (hunk 3 above)
Comment (vagelisp) [reply wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss]: Done

File: wp-content/themes/pub/wporg-learn-2020/css/customiser.css (hunk 4 above)
Comment (vagelisp) [reply wp-content/themes/pub/wporg-learn-2020/css/customiser.css]: Yes, you were right. I missed that bit, I made a mess on my local and removed it and then I didn't add it back. Anyway, I added this in the override file. Let me know what you think.

Thanks.

**Comment by vagelisp** [comment]
Hello  @adamwoodnz, let me know what you think about the tweaks.
Thanks.

File: wp-content/themes/pub/wporg-learn-2020/css/vendor/_overrides.scss [hunk 5]
```diff
@@ -536,3 +536,33 @@ p {
...
+details[open] summary::before {
+	content: "\025BE";
+}
+
+details summary {
+	display: inline-block;
+	font-weight: bold;
+	cursor: pointer;
+	line-height: 2.8;
+}
```
Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2020/css/vendor/_overrides.scss]: This appears to be for a Sensei thing so maybe move to `_sensei.scss`?

Screen Shot 2022-10-31 at 2 19 50 PM

File: wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss [hunk 6]
```diff
@@ -1,31 +1,173 @@
...
-			padding: 0;
-		}
-	}
+  &:not(.post-type-archive-course):not(.tax-course-category) {
+    section {
+      padding: 0;
+    }
+  }
+
+    .course {
```
Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss:8]: Indentation is out

```suggestion
  .course {
```

File: wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss [hunk 7]
```diff
@@ -1,31 +1,173 @@
...
+        margin-bottom: 1rem;
+      }
+
+      p {
+        font-size: 16px;
+      }
+
+      li {
+        font-size: 16px;
+      }
```
Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss]: Sorry I missed this the first time, but I think the way we've split the styles here has resulted in a lot of repetition, where these rules previously grouped elements with the same treatment.

I suggest replacing this whole block with the original styles from the customizer but with the `.sensei` parent selector removed:

```
.course:not(.post-type-archive-course) article.course:first-child,
.lesson #main-content {
	padding: 4rem 0;
}

.course article.course header,
.lesson #main-content header {
	border-bottom: 1px solid #eee;
	margin-bottom: 1rem;
}

.lesson .bbp-breadcrumb {
	display: none;
}

.lesson #main-content header {
	padding-bottom: 1rem;
}

.course article.course p,
.course article.course li,
.lesson #main-content table {
	font-size: 16px;
}

.course article.course .wp-block-image,
.lesson #main-content .wp-block-image,
.course article.course .wp-block-media-text,
.lesson #main-content .wp-block-media-text {
	margin: 2rem 0;
}

.course article.course hr,
.lesson #main-content hr {
	margin: 2rem 0;
}

.lesson #main-content nav.post-entries {
	width: 100%;
}

.lesson #main-content nav.post-entries div,
.quiz #main-content nav.post-entries div {
	display: block;
	width: 45% !important;
	padding: 1rem;
	margin: 1rem 0;
	font-style: italic;
}

.lesson #main-content nav.post-entries div.fl,
.quiz #main-content nav.post-entries div.fl {
	padding-left: 0;
}

.lesson #main-content nav.post-entries div.fr,
.quiz #main-content nav.post-entries div.fr {
	padding-right: 0;
	text-align: right;
}

.lesson #main-content .sensei-breadcrumb, 
.quiz #main-content .sensei-breadcrumb {
	padding: 1rem;
}
```

**Comment by adamwoodnz** [comment]
I think we might still be missing the very first rule from the customiser, maybe add to a new file `css/base/_table.scss`:

```
thead th {
	color: #555d66;
}
```

**Review by adamwoodnz — CHANGES_REQUESTED** [review]
Thanks for the updates! Couple more comments sorry, then we should be good to merge.

**Comment by vagelisp** [comment]
No probs, I ll get these done later today. 
Thanks for the feedback.

**Review by adamwoodnz — APPROVED** [review]
Thanks @vagelisp! I had to make a minor change to `_sensei.scss` (sorry my feedback must have been a little confusing), so I pushed these changes into a new PR which is now merged 🎉

**Comment by vagelisp** [comment]
That's great thanks.

**Pull request closed without merging**